core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -1235,7 +1235,7 @@ class CivilizationInfo : IsPartOfGameInfoSerialization {
             Stat.Culture -> policies.storedCulture
             Stat.Science -> {
                 if (tech.currentTechnology() == null) 0
-                else tech.remainingScienceToTech(tech.currentTechnology()!!.name)
+                else tech.researchOfTech(tech.currentTechnology()!!.name)
```

Comment:
Pretty sure this is how it should be, but meed to double check nothing is dependent on this